### PR TITLE
SODA Infrastructure Manager (delfin) Design spec

### PR DESCRIPTION
**Pull request by NajmudheenCT**
**What type of PR is this?**

> /kind design

**What this PR does / why we need it**:
Brings out the design document for SODA Infrastructure manager 
**Which issue(s) this PR fixes**:
NA

**Test Report Added?**:
NA
**Test Report**:
NA

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by NajmudheenCT** [comment]
@skdwriting , resolved conflict . please check

File: specs/SIM/SODA_InfrastructureManagerDesign.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+#### Storage Discovery:-
+User should be able to add supported device models to the system for discovery
+#### Storage Monitoring and Alerts: 
+Get basic storage information, hardware information, logical resources information, alert information , performance information  and configure resources and services in the storage device. 
+
+### Feature Requirements
+#### Requirement Analysis
+Analyzed Open Source monitoring solution.
+
+1.	Stor2rrd : [Analysis](https://drive.google.com/open?id=1yJn6wcji24NvObbrxvfSVzmo6--QGSI_v5P8n4Kcc8M),
```
Comment (kumarashit) [inline specs/SIM/SODA_InfrastructureManagerDesign.md:38]: Can we move it to some other place? Drive may not be accessible for all

Comment (NajmudheenCT) [reply specs/SIM/SODA_InfrastructureManagerDesign.md:38]: I hope we can remove this link.. there is no relevant  information to support this design.

Comment (NajmudheenCT) [reply specs/SIM/SODA_InfrastructureManagerDesign.md:38]: @kumarashit Need your suggestion on this

Comment (kumarashit) [reply specs/SIM/SODA_InfrastructureManagerDesign.md:38]: Yes, please remove this link. IMO, just mention it and we can give the link of the homepage

Comment (NajmudheenCT) [reply specs/SIM/SODA_InfrastructureManagerDesign.md]: done

Comment (kumarashit) [reply specs/SIM/SODA_InfrastructureManagerDesign.md]: Ok thanks

**Pull request merged**